type-definitions/most.js.flow

```diff
@@ -0,0 +1,410 @@
+declare export function generate<A>(g: CreateGenerator<A>, ...args: Array<any>): Stream<A>;
+
+declare export function reduce<A, B>(f: (b: B, a: A) => B, b: B, s: Stream<A>): Promise<B>;
+declare export function observe<A>(f: (a: A) => any, s: Stream<A>): Promise<any>;
+declare export function forEach<A>(f: (a: A) => any, s: Stream<A>): Promise<any>;
+declare export function drain<A>(s: Stream<A>): Promise<any>;
+
+declare export function subscribe<A>(subscriber: Subscriber<A>, s: Stream<A>): Subscription<A>;
+
+declare export function constant<A, B>(b: B, s: Stream<A>): Stream<B>;
+declare export function map<A, B>(f: (a: A) => B, s: Stream<A>): Stream<B>;
+declare export function tap<A>(f: (a: A) => any, s: Stream<A>): Stream<A>;
+declare export function ap<A, B>(fs: Stream<(a: A) => B>, as: Stream<A> ): Stream<B>;
+declare export function chain<A, B>(f: (a: A) => Stream<B>, s: Stream<A>): Stream<B>;
+declare export function flatMap<A, B>(f: (a: A) => Stream<B>, s: Stream<A>): Stream<B>;
+declare export function join<A>(s: Stream<Stream<A>>): Stream<A>;
+declare export function switchLatest<A>(s: Stream<Stream<A>>): Stream<A>;
+
+declare export function continueWith<A>(f: (a: any) => Stream<A>, s: Stream<A>): Stream<A>;
+declare export function concatMap<A, B>(f: (a: A) => Stream<B>, s: Stream<A>): Stream<B>;
+declare export function mergeConcurrently<A>(concurrency: number, s: Stream<Stream<A>>): Stream<A>;
+
+declare export function merge<A>(...ss: Array<Stream<A>>): Stream<A>;
+declare export function mergeArray<A>(streams: Array<Stream<A>>): Stream<A>;
+
+declare export function combine<A, B, R>(
+  fn: (a: A, b: B) => R,
+  a: Stream<A>,
+  b: Stream<B>
+): Stream<R>;
+declare export function combine<A, B, C, R>(
+  fn: (a: A, b: B, c: C) => R,
+  a: Stream<A>,
+  b: Stream<B>,
+  c: Stream<C>
+): Stream<R>;
+declare export function combine<A, B, C, D, R>(
+  fn: (a: A, b: B, c: C, d: D) => R,
+  a: Stream<A>,
+  b: Stream<B>,
+  c: Stream<C>,
+  d: Stream<D>
+): Stream<R>;
+declare export function combine<A, B, C, D, E, R>(
+  fn: (a: A, b: B, c: C, d: D, e: E) => R,
+  a: Stream<A>,
+  b: Stream<B>,
+  c: Stream<C>,
+  d: Stream<D>,
+  e: Stream<E>
+): Stream<R>;
+
+declare export function combineArray<A, B, R>(
+  fn: (a: A, b: B) => R,
+  streams: [Stream<A>, Stream<B>]
+): Stream<R>;
+declare export function combineArray<A, B, C, R>(
+  fn: (a: A, b: B, c: C) => R,
+  streams: [Stream<A>, Stream<B>, Stream<C>]
+): Stream<R>;
+declare export function combineArray<A, B, C, D, R>(
+  fn: (a: A, b: B, c: C, d: D) => R,
+  streams: [Stream<A>, Stream<B>, Stream<C>, Stream<D>]
+): Stream<R>;
+declare export function combineArray<A, B, C, D, E, R>(
+  fn: (a: A, b: B, c: C, d: D, e: E) => R,
+  streams: [Stream<A>, Stream<B>, Stream<C>, Stream<D>, Stream<E>]
+): Stream<R>;
+declare export function combineArray<V, R> (
+  fn: (...items: V[]) => R,
+  items: Stream<V>[]
+): Stream<R>;
```

Comment:
Whatever we decide about any vs V for the method form of combineArray, we can apply here, too.